app/templates/cp_nigeria/help_pages/help_page.html

```diff
@@ -0,0 +1,98 @@
+{% extends 'cp_nigeria/base_cpn.html' %}
+{% load custom_filters %}
+{% load crispy_forms_tags %}
+{% load static %}
+{% load i18n %}
+
+{% block head_block %}
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.6.0/dist/leaflet.css" crossorigin=""
+          integrity="sha512-xwE/Az9zrjBIphAcBb3F6JVqxf46+CDLwfLMHloNu6KEQCAWi6HcDUbeOfBIptF7tcCzusKFjFw2yuvEpDL9wQ=="/>
+    <link rel="stylesheet" href="{% static 'css/map.css' %}" type="text/css">
+{% endblock head_block %}
+
+{% block start_body_scripts %}
+    <script src="https://unpkg.com/leaflet@1.6.0/dist/leaflet.js" crossorigin=""
+            integrity="sha512-gZwIG9x3wUXg2hdXF6+rVkLF/0Vi9U8D2Ntg4Ga5I5BZpVkVxlJWbSQtXPSiUTtC0TjtGOmxa1AJPuV0CPthew==">
+    </script>
+{% endblock %}
+
+{% block title %}{% translate "Help" %}{% endblock title %}
+
+{% block content %}
+
+
+<main>
+    <section class="header">
+        <div>
+            <div class="header__left"></div>
+            <h1 class="header__title">{% translate "Help" %}</h1>
+            <div class="header__back"></div>
+        </div>
+    </section>
+    <section class="project-setup two-columns">
+        <div class="col-md-3">
+            <nav style="position: fixed; background-color:#E6F7EE">
+                <ul class="nav flex-column" id="help_sidebar">
+                    {% for step, verbose in steps %}
+                    <li class="nav-item">
+                        <a class="nav-link" href="#section-{{ step }}">{{ verbose }}</a>
+                    </li>
+                    {% endfor %}
+                </ul>
+            </nav>
+        </div>
+		<div class="col-md-7" id="help_content">
+            <div>
+			{% for step, verbose in steps %}
+			<div id="section-{{ step }}" style="padding-top: 1rem; padding-bottom: 1rem">
+				<h2>{{ verbose }}</h2>
+				<div class="section-content" data-step="{{ step }}"></div>
+			</div>
+			{% endfor %}
+            </div>
+		</div>
+    </section>
+</main>
+
+
+{% endblock content %}
+
+{% block end_body_scripts %}
+
+<script>
+    function loadHelpContent(step) {
```

Comment:
Nice that if the steps order changes, the help order also changes. It would have been possible to display this without extra ajax call using directly templates